pkg/symbol/addr2line/add2line.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package addr2line
+
+import "errors"
+
+var ErrSymbolNotFound = errors.New("symbol not found")
```

Comment:
I thought it would be shared between different files of the same package. But I've changed things afterwards. I think I can even remove the sentinel error now. I'll look into it.